### PR TITLE
[DEV-2551] balance monitoring

### PR DESCRIPTION
**Pull request by bernd2022**

### CODE REVIEW

File: src/subdomains/monitoring/services/monitoring.service.ts
Comment (davidleomay) [inline src/subdomains/monitoring/services/monitoring.service.ts:1]: Every 5 seconds might be a bit often?

Comment (bernd2022) [reply src/subdomains/monitoring/services/monitoring.service.ts:1]: These "cron" functions will be called in the new implementation, if there are new transactions incoming. Therefore the "cron" will be removed completely

Comment (davidleomay) [reply src/subdomains/monitoring/services/monitoring.service.ts:1]: Which implementation do you mean?

Comment (bernd2022) [reply src/subdomains/monitoring/services/monitoring.service.ts:1]: The implementation in my development workspace coming soon in this PR ;-)

Comment (davidleomay) [reply src/subdomains/monitoring/services/monitoring.service.ts:1]: Ok, is this also including the channel balances? 😉

Comment (bernd2022) [reply src/subdomains/monitoring/services/monitoring.service.ts:1]: Yes!

Comment (davidleomay) [reply src/subdomains/monitoring/services/monitoring.service.ts:1]: Cool 😉

**Pull request merged**